libexec/pool/registry/registry-update

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+### registry-update -- update pool list & pool metadata
+### Usage: pool update <root> [<args>]
+###    or: pool update [<args>]
+###
+### Note: for updating pool list, always run `pool registry merge` first
```

Comment:
Any chance we could automatically run this first? Seems like an easy step to forget.